TABS/tabs.js

```diff
@@ -1,37 +1,34 @@
 //tabs
 $(document).ready(function(){
 	$('.tabs__controls-link').on('click', function(e) {
-  		e.preventDefault();
-   		var item=$(this).closest('.tabs__controls-item');
-   		var contentItem=$('.tabs__item');
-   		var position=item.index();
+		e.preventDefault();
+   		var item = $(this).closest('.tabs__controls-item');
+   		var contentItem = $('.tabs__item');
+   		var position = item.index();
+		
    		contentItem.eq(position)
    			.addClass('tabs__item_active')
    			.siblings()
-   			.removeClass('tabs__item_active')
+   			.removeClass('tabs__item_active');
    		item
    			.addClass('tabs__controls-item_active')
    			.siblings()
-   			.removeClass('tabs__controls-item_active')
+   			.removeClass('tabs__controls-item_active');
   	});
-});//ready
-
-
-
-//tabs with no worry about the order in the list
-$(document).ready(function(){
```

Comment:
You need to have one $(document).ready function and store all code inside that function.